src/compiler/utilities.ts

```diff
@@ -6288,6 +6290,21 @@ export function getPossibleOriginalInputExtensionForExtension(path: string) {
         [Extension.Tsx, Extension.Ts, Extension.Jsx, Extension.Js];

```

Comment:
tangent: would it make any sense to limit this to avoid returning js/jsx for `path` with ts/tsx?